command/file.go

```diff
@@ -33,24 +33,28 @@ func CopyDir(src, dst string, isOnlyContent bool) error {
 }
 
 // RemoveDir ...
-func RemoveDir(dirPth string) error {
-	if exist, err := pathutil.IsPathExists(dirPth); err != nil {
-		return err
-	} else if exist {
-		if err := os.RemoveAll(dirPth); err != nil {
+func RemoveDir(dirPths ...string) error {
+	for _, dirPth := range dirPths {
+		if exist, err := pathutil.IsPathExists(dirPth); err != nil {
```

Comment:
pls remove this check, as in case of "remove" the main goal is to not to exists the dirs, and if it is already not exists then RemoveAll will not fail at all or if it is existing the it will remove it. unnecessary step here.

https://golang.org/pkg/os/#RemoveAll